kornia/cli.py

```diff
@@ -0,0 +1,101 @@
+"""Command-line interface for Kornia."""
+
+import argparse
+import logging
+from dataclasses import dataclass
+from pathlib import Path
+
+import torch
+
+from kornia.contrib import FaceDetector
+
+logging.basicConfig(level=logging.DEBUG, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class ExportOnnxModelConfig:
+    """Configuration for exporting a model to ONNX format."""
+
+    model: torch.nn.Module
+    img: torch.Tensor
+    output_path: Path
+    input_names: list[str]
+    output_names: list[str]
+    dynamic_axes: dict[str, dict[int, str]]
+
+
+def export_onnx_model(config: ExportOnnxModelConfig) -> None:
+    """Export a model to ONNX format."""
+    torch.onnx.export(
+        config.model,
+        config.img,
+        config.output_path,
+        input_names=config.input_names,
+        output_names=config.output_names,
+        dynamic_axes=config.dynamic_axes,
+    )
+
+    if config.output_path.exists():
+        logger.info("Model exported to %s", config.output_path)
+    else:
+        logger.error("Error exporting model to %s", config.output_path)
+
+
+def export_onnx_model_config_resolver(model_name: str, output_path: Path) -> ExportOnnxModelConfig:
+    """Resolve the configuration for exporting a model to ONNX format."""
+    if model_name == "kornia/yunet":
```

Comment:
i would assume, any model without the explicit module is from kornia
```suggestion
    if model_name in {"kornia/yunet", "yunet"}:
```

Reply:
once matured, this should be an enum to ve less error prone